src/GitHubStatusDisplay.js

```diff
@@ -264,6 +264,7 @@ export default class BuildHistoryDisplay extends Component {
       "/",
       "_"
     )}.json`;
+    console.log("JSON URL IS", jsonUrl);
```

Comment:
Should we leave this in for when we ship?

Reply:
whoops. Didn't save :|